### PR TITLE
Clarify nrpe, nagios, and check_nrpe relationship

### PR DESCRIPTION
**Pull request by thenewwazoo**

### CODE REVIEW

File: nagios/map.jinja [hunk 1]
```diff
@@ -83,21 +86,22 @@
...
 {% set nrpe = salt['grains.filter_by']({
     'default': {
-        'plugin_dir': '/usr/lib/nagios/plugins',
+        'plugins_dir': '/usr/lib/nagios/plugins',
     },
     'Arch': {
-        'plugin': 'nagios-nrpe-plugin',
-        'server': 'nagios-nrpe',
+        'plugins': 'monitoring-plugins',
+        'server':  'nagios-nrpe',
```
Comment (cdarwin) [inline nagios/map.jinja]: whitespace

File: nagios/map.jinja [hunk 2]
```diff
@@ -83,21 +86,22 @@
...
+        'cfg_dir': '/etc/nagios/nrpe.d',
+        'plugins': 'nagios-plugins-basic',
+        'server':  'nagios-nrpe-server',
         'service': 'nagios-nrpe-server',
     },
     'RedHat': {
-        'plugin': 'nagios-plugins-all',
-        'server': 'nrpe',
+        'plugins': 'nagios-plugins-all',
+        'server':  'nrpe',
```
Comment (cdarwin) [inline nagios/map.jinja]: whitespace

Comment (thenewwazoo) [inline nagios/map.jinja:104]: Not sure I follow. What whitespace is incorrect?

**Comment by cdarwin** [comment]
+1
lgtm

File: nagios/map.jinja (hunk 2 above)
Comment (cdarwin) [inline nagios/map.jinja:104]: please, disregard

**Pull request merged**